conf/application.conf

```diff
@@ -154,4 +154,9 @@ urls {
   signOut = "/register-for-exchange-of-information-frontend"
 }
 
-json.countries = "countries.json"
\ No newline at end of file
+json.countries = "countries.json"
+keys {
+  enrolmentKey.mdr = "HMRC-MDR-ORG"
+}
+
+urls.mandatory-disclosure-rules-frontend = "http://localhost:10018/mandatory-disclosure-rules-frontend"
```

Comment:
@shaileshp0110 Can we place this inside urls block
urls {
  login         = "http://localhost:9949/auth-login-stub/gg-sign-in"
  loginContinue = "http://localhost:10018/mandatory-disclosure-rules-frontend"
  signOut       = "http://localhost:9025/gg/sign-out"
}